catalog/github/webhook.js

```diff
@@ -7,58 +7,106 @@ var request = require('request');
  *  @param {string} username - github username
  *  @param {string} accessToken - github access token
  *  @param {string} events - list of the events the webhook should fire on
+ *  @param {string} endpoint - The endpoint the openwhisk deployed on
  *  @return {object} whisk async
  */
 function main(params) {
-  var username = params.username;
-  var repository = params.repository;
-  var accessToken = params.accessToken;
+    var username = params.username;
+    var repository = params.repository;
+    var accessToken = params.accessToken;
 
```

Comment:
@lzbj I believe what Rodric is suggesting is that, instead of setting the default endpoint value in code, you should specify the value when the package is created.  Please see [this example](https://github.com/openwhisk/openwhisk/blob/master/catalog/installWeather.sh#L15) in the script which installs the weather package. Then your code can just refer to `params.endpoint` with confidence that if the parameter was not explicitly set when creating the feed, then the value will already be the default.

@rabbah Please confirm this was your intention.